apps/game_backend/priv/repo/migrations/20240429180719_add_google_user_assoc_to_users.exs

```diff
@@ -0,0 +1,10 @@
+defmodule GameBackend.Repo.Migrations.AddGoogleUserAssocToUsers do
+  use Ecto.Migration
+
+  def change do
+    alter table(:users) do
+      add :google_user_id, references(:google_users, on_delete: :delete_all)
+      add :profile_picture, :string
```

Comment:
What is this supposed to store?

Reply:
Just a string. i.e.: "picture_1", "picture_2", etc. It's like characters.
At least for now.